src/test/java/uk/ac/ebi/eva/pipeline/jobs/steps/VepInputGeneratorStepTest.java

```diff
@@ -93,5 +109,6 @@ public void shouldGenerateVepInput() throws Exception {
 
         assertTrue(vepInput.exists());
         assertEquals("20\t60343\t60343\tG/A\t+", readFirstLine(vepInput));
+        assertEquals(EXPECTED_NON_ANNOTATED_VARIANTS, getLines(new FileInputStream(vepInput)));
     }
```

Comment:
yes, the dump only contains one study, so the result is the same. The second test is more to check that the parameters validator allow an empty string as studyId. I will change the test name to highlight that.